examples/js/loaders/GLTFLoader.js

```diff
@@ -110,10 +113,35 @@ THREE.GLTFLoader = ( function () {
 
 		},
 
+		register: function ( callback ) {
+
+			if ( ! this.pluginCallbacks.includes( callback ) ) {
```

Comment:
Let's avoid array methods missing from IE11 if we can, see https://github.com/mrdoob/three.js/issues/19293. It is simpler if the only polyfill GLTFLoader requires is the Promise API.

Reply:
I was missing IE11 doesn't support `includes()`. Replaced with `indexOf()`, thanks.